www/common/common-ui-elements.js

```diff
@@ -623,13 +623,19 @@ define([
         var appType = (common.getMetadataMgr().getMetadata().type || 'pad').toUpperCase();
         data = data || {};
         if (!callback && data.callback) { callback = data.callback; }
+
+        function createIconButton(iconClasses, buttonClasses, title, text, ariaLabel) {
```

Comment:
The new name "createMenuItemButton" doesn't seem right to me either because some button are not menu entries.
What this function does it to create a base button that can be customized for each use-case below. Maybe just "makeButton"?